src/framework/pybind_json.hpp

```diff
@@ -374,6 +405,25 @@ py::object AerToPy::from_avg_data(const AER::AverageData<T> &avg_data) {
   return std::move(d);
 }
 
+template<typename T> 
+py::object AerToPy::from_avg_data(const AER::AverageData<matrix<T>> &avg_data) {
+  py::dict d;
+  d["value"] = AerToPy::array_from_matrix(avg_data.mean());
+  if (avg_data.has_variance()) {
+    d["variance"] = AerToPy::array_from_matrix(avg_data.variance());
+  }
+  return d;
```

Comment:
I think `std::move()` here will avoid a copy as the return value of the function and `py::dict` are different objects so he can't figure out how to do RVO (Return Value Optimization)

Reply:
you're right on all